scripts/linux/install-kubernetes.sh

```diff
@@ -13,16 +13,42 @@ while true; do
   esac
 done
 
+ansible_debug="$1"
+
+echo "Ansible debug: $ansible_debug"
+
+verbose_flag=
+if [ "$ansible_debug" = 'on' ]; then
+    verbose_flag="-vv"
+fi
+
+echo "Verbose verbose_flag: $verbose_flag"
+
 echo "Ensure the Docker service is enabled and running"
+
+exit
+
 systemctl enable docker
 systemctl restart docker
 
 inventory="/etc/"$inventory
 
 echo "Running Ansible playbooks against $inventory inventory"
-docker run --rm \
-    -v /vagrant/ansible:/etc/ansible \
-    -v /vagrant/ansible/playbooks/files/tls:/opt/tls/self_signed \
-    --net=host \
-    ferrarimarco/open-development-environment-ansible:2.7.12-alpine \
-    /bin/sh -c "ansible-galaxy install -r /etc/ansible/requirements.yml && ansible-playbook -i $inventory /etc/ansible/playbooks/kubernetes.yml && ansible-playbook -i $inventory /etc/ansible/playbooks/openssl-self-signed-certificate.yml"
+
+if [ "$ansible_debug" = 'on' ]; then
```

Comment:
Double quote "on" for consistency

Reply:
done, we should do it also in bootstrap-kubernetes-master.sh and bootstrap-kubernetes-minion.sh